crates/engine/schema/src/lib.rs

```diff
@@ -258,23 +258,25 @@ impl std::fmt::Debug for Schema {
 /// Defines how a scalar should be represented and validated by the engine. They're almost the same
 /// as scalars, but scalars like ID which have no own data format are just mapped to String.
 /// https://the-guild.dev/graphql/scalars/docs
-#[derive(
-    Debug, Clone, Copy, PartialEq, Eq, strum::Display, strum::EnumString, serde::Serialize, serde::Deserialize,
-)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
 pub enum ScalarType {
     String,
     Float,
     Int,
     BigInt,
-    JSON,
     Boolean,
+    Any,
```

Comment:
This seems like a potentially confusing name given there's a literal `Any` scalar which has a specific format.

`Unknown` or something might be less ambiguous?

Reply:
what? which `Any` are you talking about?